linux/user-and-file-management/linux-intro/what-is-linux.md

```diff
@@ -0,0 +1,39 @@
+---
+author: kapnobatai136
+
+aspects:
+  - introduction
+
+type: normal
+
+category: must-know
+
+tags:
+  - introduction
+  - linux
+  - operating-system
+
+---
+
+# What's Linux?
+
+---
+## Content
+
+To understand what Linux is, let's go back to the very basics of computing.
+
+To complete even the most basic operation, something must facilitate the communication between you and the hardware. 
+
+That's what an **operating system** does. It makes it possible for you to read this.
+
+Along with Windows, Android, or MacOS, Linux is an **operating system**. You can find it in many places. From your computer, to your car and refrigerator, there's a high probability that they run on Linux.
+
+---
+## Practice
+
+Operating systems facilitate communication between you and the ???.
+
+* hardware
+* software
+* middleware
+* wares
```

Comment:
This is what I could come up to match my [suggestion from above](https://github.com/enkidevs/curriculum/pull/2096/files#r424719171) :D

```suggestion
Linux is an ???.

* operating system
* ordering system
* operating software
* ordering software
```